doc/nrf/libraries/others/date_time.rst

```diff
@@ -30,6 +30,10 @@ When this library retrieves the date-time information, it is fetched in the foll
 #. If the time information obtained from the nRF91 Series modem is not valid and the :kconfig:option:`CONFIG_DATE_TIME_NTP` option is set, the library requests time from an NTP server.
 #. If the NTP time request does not succeed, the library tries to request time information from a different NTP server, before it fails.
 
+If all time requests fail and the :kconfig:option:`CONFIG_DATE_TIME_RETRY_COUNT` Kconfig option is greater than zero, the entire process listed above will be retried.
+The interval before each retry is defined by the :kconfig:option:`CONFIG_DATE_TIME_UPDATE_INTERVAL_SECONDS` Kconfig option, and the :kconfig:option:`CONFIG_DATE_TIME_RETRY_COUNT` Kconfig option sets how many consecutive retries are attempted before giving up.
```

Comment:
I assume this has changed mistakenly to wrong Kconfig?

```suggestion
The interval before each retry is defined by the :kconfig:option:`CONFIG_DATE_TIME_RETRY_INTERVAL_SECONDS` Kconfig option, and the :kconfig:option:`CONFIG_DATE_TIME_RETRY_COUNT` Kconfig option sets how many consecutive retries are attempted before giving up.
```

Reply:
Oh, nice catch!